core/assets/bundles/bundle_ko.properties

```diff
@@ -599,8 +618,8 @@ setting.difficulty.insane = 미침
 setting.difficulty.name = 난이도 : 
 setting.screenshake.name = 화면 흔들기
 setting.effects.name = 화면 효과
-setting.destroyedblocks.name = Display Destroyed Blocks
-setting.conveyorpathfinding.name = Conveyor Placement Pathfinding
+setting.destroyedblocks.name = 부서진 블럭 표시
+setting.conveyorpathfinding.name = 컨베이어 설치 보조 기능
 setting.sensitivity.name = 컨트롤러 감도
 setting.saveinterval.name = 저장 간격
 setting.seconds = 초
```

Comment:
This needs a `{0}` somewhere, the English version is

`setting.seconds = {0} Seconds`

with `{0}` being the number of seconds.

Reply:
oh.

Reply:
thank for your advice